app/models/miq_expression.rb

```diff
@@ -118,6 +119,8 @@ class MiqExpression
     ontap_raid_group_extents
     ontap_storage_systems
     ontap_storage_volumes
+    openscap_results
+    openscap_rule_results
```

Comment:
I tested this with out #6839, that is with evm:db:reset on this pr alone. It works fine - the fields related to this table
are not shown. However now that you mention it I cannot guarantee that every other place that works with miq expression will not break.